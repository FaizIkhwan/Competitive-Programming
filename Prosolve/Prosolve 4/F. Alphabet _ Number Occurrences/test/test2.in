2
hello boys
r u prepared for prosolve 4.0?!!