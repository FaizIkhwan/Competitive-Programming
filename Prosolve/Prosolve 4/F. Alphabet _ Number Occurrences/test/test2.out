Alphabet: 9, Digit: 0
Alphabet: 21, Digit: 2
